src/MSI/Product.wxs

```diff
@@ -153,7 +153,8 @@ Licensed under the MIT license. See LICENSE file in the project root for full li
                 <File Source="..\AccessibilityInsights.VersionSwitcher\bin\Release\net472\AccessibilityInsights.SetupLibrary.dll" Id ="version_switcher_setuplibrary"/>
                 <File Source="..\AccessibilityInsights.VersionSwitcher\bin\Release\net472\AccessibilityInsights.Win32.dll" Id ="version_switcher_win32"/>
                 <File Source="..\AccessibilityInsights.VersionSwitcher\bin\Release\net472\Microsoft.Deployment.WindowsInstaller.dll" Id ="version_switcher_deployment"/>
-                <File Source="..\AccessibilityInsights.VersionSwitcher\bin\Release\net472\Newtonsoft.Json.dll" Id ="version_switcher_newtonsoft"/>
+                <!-- We sign Newtonsoft.Json.dll in the main AccessibilityInsights project. To avoid duplicate signing, we reuse it below. -->
+                <File Source="..\AccessibilityInsights\bin\Release\net472\Newtonsoft.Json.dll" Id ="version_switcher_newtonsoft"/>
```

Comment:
This looks good to me. We have a similar issue with Microsoft.Deployment.WIndowsInstaller.dll (which is signed by WiX, not by Microsoft)

Reply:
Good catch! Here I was assuming everything prefixed with Microsoft.* would be signed by Microsoft 🙄 I will raise another PR for that.